WordPress/src/main/java/org/wordpress/android/util/DialogUtils.java

```diff
@@ -11,22 +14,55 @@
 import org.wordpress.android.widgets.OpenSansEditText;
 import org.wordpress.android.widgets.WPTextView;
 
-public class DialogUtils {
-    public static void showMyProfileDialog(Context context,
-                                           String title,
-                                           String initialText,
-                                           String hint,
-                                           boolean isMultiline,
-                                           final Callback callback) {
-        LayoutInflater layoutInflater = LayoutInflater.from(context);
+public class DialogUtils extends DialogFragment {
+
+    private static final String TITLE_TAG = "TITLE";
+    private static final String INITIAL_TEXT_TAG = "INITIAL-TEXT";
+    private static final String HINT_TAG = "HINT";
+    private static final String IS_MULTILINE_TAG = "IS-MULTILINE";
+
+    private static Context mContext;
```

Comment:
Hmm, please avoid using globals like that. In this case, there is case, there's even no need to keep a referece to Context around. See below for a way to revise this code. Cheers!